masterlist.yaml

```
@@ -10729,6 +10729,11 @@ plugins:
       - <<: *alreadyInX
         subs: [ 'Rustic Soulgems Unsorted GIST' ]
         condition: 'file("RUSTIC SOULGEMS - Unsorted( - ESL)? - GIST\.esp")'
+    tag:
+      - Graphics
```

Lists with a single entry can be put on one line.
```suggestion
    tag: [ Graphics ]
```